core/templates/pages/story-editor-page/modal-templates/new-chapter-title-modal.template.html

```diff
@@ -16,9 +16,9 @@ <h3>
     </span>
   </div>
   <div class="modal-footer">
+    <button class="btn btn-secondary" ng-click="cancel()">Cancel</button>
```

Comment:
Wasn't this 'Cancel' already on the left? Ditto below.

Reply:
Hi @aks681, I made this change where ever I saw the problematic pattern (ie. cancel/btn-secondary first followed by submit/btn-primary).

I have just verified that 'Cancel' is on the right in develop.
![image](https://user-images.githubusercontent.com/11008603/76095283-0808c580-5fea-11ea-9565-a8618e51c819.png)